pkg/stanza/operator/parser/syslog/syslog.go

```diff
@@ -317,6 +352,15 @@ var severityText = [...]string{
 
 var severityField = entry.NewAttributeField("severity")
 
+func postprocessWithoutPriHeader(e *entry.Entry) error {
+	_, ok := entry.NewAttributeField("timestamp").Delete(e)
+	if !ok {
+		return fmt.Errorf("failed to cleanup timestamp")
+	}
+
+	return nil
```

Comment:
Would be good to extract this code into a function like `cleanupTimestamp` or something, and use the function in both `postprocessWithoutPriHeader` and `postprocess` to avoid duplication.